config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-presubmits.yaml

```diff
@@ -21,6 +21,9 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-openstack
+      test-tab-name: pr-make
```

Comment:
spot-check: 

the tab name doesn't seem to be taking effect

I instead see https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-openstack#pull-cluster-api-provider-openstack-make

Reply:
This should be `testgrid-tab-name` according to: https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration